packages/pinpin/README.md

```diff
@@ -30,6 +31,22 @@ npm run start
 
 ## Running on Heroku
 
-A Procfile is added to the root of the monorepo to define the `pinpin` process.
-- we have a web3-storage team, so run it there.
-- set 
\ No newline at end of file
+Deploy using the `herkou` command. Install the `heroku` cli and log in with `heroku login`. A Procfile is added to the root of the monorepo to define the `pinpin` process. It's a worker that should run on a single dyno. There is no `web` worker, which heroku assumes there will be, so we have to explicitly scale the dynos to make that the case. Deployment uses git, and you get `heroku` remote to push to, added to your repo automatically.
+
+Deploying from scratch looks like this:
+
+```console
+# create the app as a resource under the web3-storage team, and add `heroku` as git remote
+$ heroku create --team=web3-storage web3-storage
+
+# set the env (or better, use the webui for secrets)
+$ heroku config.set FAUNA_KEY=*** PINATA_JWT=***
+
+# deploy the local pinpin branch to heroku via git (you might push from a local main branch instead.)
```

Comment:
once this PR is merged, the pinpin branch will be no more?

Reply:
i was documenting what i did, as I'm not expecting to do anything else on this or even set up CI on it for the foresable future, so i'll probably want to deploy a local branch name again if there is a next time.